wire-ios-sync-engine/Source/Data Model/Conversation+ReadReceiptMode.swift

```diff
@@ -49,9 +49,17 @@ public extension ZMConversation {
         guard let conversationId = remoteIdentifier?.transportString()
         else { return completion(.failure(ReadReceiptModeError.noConversation)) }
 
+        let path: String
+        if apiVersion >= .v8 {
+            guard let domain else { return completion(.failure(ReadReceiptModeError.noConversation)) }
```

Comment:
suggestion: also here it'd be good to fallback on the local domain.

Reply:
done